### PR TITLE
Test development

### PR DESCRIPTION
**Pull request by Sithyphus**
I incorporated the changes you suggested from the base-docker-support pull request but of course just let me know what may or may not need changing.

### CODE REVIEW

File: test-files/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Transformer Unit Tests
+This folder holds the unit tests [TravisCI](https://travis-ci.org/) will run in it's build. 
+These are basic functional unit tests, checks for output formatting and typing. 
+All tests are written in and utilizing [pytest](https://docs.pytest.org/en/latest/). 
+In addition to this, [pylint](https://www.pylint.org/) will also be deployed. 
+Look into our organization's repository for our [pylint protocols](https://github.com/AgPipeline/Organization-info).
+
+### Running the tests
+Upon submitting a pull request Travis will build and run the testing modules automatically and will return a report with all passing or failing code.
```
Comment (Chris-Schnaufer) [inline test-files/README.md]: '... with **a** passing or ...' - should 'all' be changed to 'a'?

File: test-files/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ```sh
+    python -m pytest -v
+    ```
+    or
+    ```sh
+    python3 -m pytest -v
+    ```
+
+### Requirements 
+There are no additional requirements or dependancies if not running these tests locally, if however these are to be run before deploying travis the following are required.
```
Comment (Chris-Schnaufer) [inline test-files/README.md]: Perhaps change to "These are the requirements needed when running the tests locally"

File: test-files/test_transformer.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+"""Testing files for transformer.py
+"""
+
+#Import transformer.py and imbedded modules
```
Comment (Chris-Schnaufer) [inline test-files/test_transformer.py]: `imbedded` is archaic, change to `embedded`?

File: test-files/test_transformer.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import argparse
+import transformer
+import transformer_class
+
+#Setting up some testing classes
```
Comment (Chris-Schnaufer) [inline test-files/test_transformer.py]:
```suggestion
# Setting up some testing classes
```

File: test-files/test_transformer.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+TEST_TRANSFORMER = transformer_class.Transformer()
+PARSE = argparse.ArgumentParser()
+
+# pylint: disable=assignment-from-no-return
+def test_add_parameters():
+    """Test for add_parameters function
+    """
+
+    #Saving method call to variable
```
Comment (Chris-Schnaufer) [inline test-files/test_transformer.py]: There should be a space between the `#` and the first following character. Here and elsewhere

File: test-files/test_transformer.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    test_params = transformer.add_parameters(PARSE)
+
+    #Should return None
+    assert test_params is None
+
+def test_check_continue():
+    """Test for check continue
```
Comment (Chris-Schnaufer) [inline test-files/test_transformer.py]:
```suggestion
    """Test for check_continue function
```

File: test-files/README.md (hunk 1 above)
Comment (Sithyphus) [reply test-files/README.md]: Changed the sentence to this, I think this probably looks better

"Upon submitting a pull request Travis will build and run the testing modules automatically and return a report with passing and failing code."

Comment (Chris-Schnaufer) [reply test-files/README.md]: I like it!

**Comment by Chris-Schnaufer** [comment]
Closing pull request due to problems with TravisCI integration. Will create a new pull request once those issues are resolved.

**Pull request closed without merging**